.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,133 @@
+name: Build and Deploy From A2B API
+
+on:
+  push:
+    branches:
+      - main
+      - develop
```

Comment:
why on push and on PR ?

Reply:
for testing availability of new code to be build and deployed before merge